src/Tribe/REST/V1/EA_Messages.php

```diff
@@ -4,21 +4,44 @@ class Tribe__Events__REST__V1__EA_Messages extends Tribe__Events__REST__V1__Mess
+			'tec-rest-api-unsupported'           => __( 'Events could not be imported. The requested URL does not support The Events Calendar REST API.', 'the-events-calendar' ),
+			'tec-rest-api-disabled'              => __( 'Events could not be imported. The Events Calendar REST API is disabled on the requested URL.', 'the-events-calendar' ),
+			'tec-rest-api-bad-data'              => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but returned malformed data.', 'the-events-calendar' ),
+			'tec-rest-api-archive-header-error'  => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but there was an error while fetching the archive control data.', 'the-events-calendar' ),
+			'tec-rest-api-archive-missing-total' => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but there was an error while fetching the total number of events.', 'the-events-calendar' ),
+			'tec-rest-api-archive-bad-total'     => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but returned malformed data in regard to the total number of events.', 'the-events-calendar' ),
+			'tec-rest-api-archive-page-error'    => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but there was an error while fetching an archive page.', 'the-events-calendar' ),
+			'tec-rest-api-archive-page-empty'    => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but returned an empty archive page.', 'the-events-calendar' ),
+			'tec-rest-api-single-event-error'    => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but there was an error while fetching the event data.', 'the-events-calendar' ),
+			'tec-rest-api-single-event-empty'    => __( 'Events could not be imported. The URL provided could be reached and has The Events Calendar REST API enabled, but returned empty event data.', 'the-events-calendar' ),
 			'no-results'                         => __( 'The requested URL does not have any upcoming and published events matching the search criteria.', 'the-events-calendar' ),
 		);
 
+		$learn_more_map = array(
+			'tec-rest-api-unsupported',
+			'tec-rest-api-disabled',
+			'tec-rest-api-bad-data',
+			'tec-rest-api-archive-header-error',
+			'tec-rest-api-archive-missing-total',
+			'tec-rest-api-archive-bad-total',
+			'tec-rest-api-archive-page-error',
+			'tec-rest-api-archive-page-empty',
+			'tec-rest-api-single-event-error',
+			'tec-rest-api-single-event-empty',
+		);
+
+		// @todo @luca add the Knowledgebase link here!
```

Comment:
Is the plan to merge and update later (I feel like you mentioned this on the weekly review meeting, but I can't remember what you said). Also, I wonder if we could create and use an `m.tri.be` link and simply update the destination URL later, once the KB post exists